recipe/run_test.bat

```diff
@@ -1,7 +1,17 @@
+@echo on
 
-set PYTHONPATH=%LIBRARY_PREFIX%\python;%PYTHONPATH%
+:: Check for QGIS executable and shortcut
+if not exist "%LIBRARY_PREFIX%\qgis\bin\qgis-bin.exe" exit /b 1
```

Comment:
Seems really close now on Windows - well done @ceholden - but I'm guessing this should read:
```
if not exist "%LIBRARY_PREFIX%\bin\qgis-bin.exe" exit /b 1
```
Since the prefix got changed...

Reply:
I just noticed this! I'm taking a stab at getting the `...\qgis.exe --postinstall` as a `post-link.bat`\`pre-unlink.bat` pair of scripts but should at least confirm the tests pass locally and update this PR with the fix...

On Mac we just need the QScintilla patch to be merged and rebuilt. The PR looks good but we're not so sure how the providers work out since it hasn't been rerendered in a while and there's builds from Azure. I think we're dealing with the same thing here when we're ready to merge (needing to set `provider` fields)